webrtc/coverage/set-session-description.txt

```diff
@@ -0,0 +1,242 @@
+Coverage Report is based on the following editor draft:
+https://w3c.github.io/webrtc-pc/archives/20170605/webrtc.html
+
+4.4.1.6 Set the RTCSessionSessionDescription
+
+  [Trivial]
+  1.  Let p be a new promise.
+
+  [Trivial]
+  2.  In parallel, start the process to apply description as described in [JSEP]
+      (section 5.5. and section 5.6.).
+
+    [Trivial]
+    1.  If the process to apply description fails for any reason, then user agent
+        MUST queue a task that runs the following steps:
+
+      [RTCPeerConnection-setLocalDescription]
+      [RTCPeerConnection-setRemoteDescription]
+      1.  If connection's [[IsClosed]] slot is true, then abort these steps.
+
+      [RTCPeerConnection-setLocalDescription]
+      [RTCPeerConnection-setRemoteDescription]
+      2.  If the description's type is invalid for the current signaling state of
+          connection as described in [JSEP] (section 5.5. and section 5.6.), then
+          reject p with a newly created InvalidStateError and abort these steps.
+
+      [RTCPeerConnection-setLocalDescription]
+      [RTCPeerConnection-setRemoteDescription]
+      3.  If the content of description is not valid SDP syntax, then reject p
+          with an RTCError (with errorDetail set to "sdp-syntax-error" and the
+          sdpLineNumber attribute set to the line number in the SDP where the
+          syntax error was detected) and abort these steps.
+
+      [Untestable]
+      4.  If the content of description is invalid, then reject p with a newly
+          created InvalidAccessError and abort these steps.
```

Comment:
This is the content of 2.1.5. The contents of 2.1.4 are incorrectly shown above in 2.1.3.